sycl/test-e2e/AddressSanitizer/nullpointer/private_nullptr.cpp

```diff
@@ -8,6 +8,7 @@
 
 // FIXME: There's an issue in gfx driver, so this test pending here.
 // XFAIL: *
+// XFAIL-TRACKER: GSD-9911
```

Comment:
I heard that we shouldn't use JIRA to track issues in syclos, am I right?

Reply:
I believe we are allowed to put JIRA numbers but not JIRA links.